client/state/analytics/selectors.js

```diff
@@ -0,0 +1,7 @@
+export function isTracking( state ) {
+	return ( state.application.continuousTracking === 'TRACKING' );
+}
```

Comment:
With the changes to the reducer I'm suggesting, this can be changed to

```js
export function isTracking( state ) {
	return state.application.continuousTracking;
}
```